requirements/base.txt

```diff
@@ -31,7 +31,7 @@ regex!=2021.10.8
 tzlocal==5.3.1
 
 #Adding cfn-lint dependency for SAM validate
-cfn-lint~=1.40.4
+cfn-lint~=1.40.2
```

Comment:
You also undid you own PR that updated cfn-lint: https://github.com/aws/aws-sam-cli/pull/8401/files